### DISCUSSION TITLE
Quasar sorting icon not showing

### DISCUSSION THREAD

**Discussion by mtn090899** [quasarframework/quasar#13178] [General - Components / Directives / etc]
Hi,

So I'm trying to make a custom header in my table utilizing qth, however the sorting icon is not showing up and it is not letting me pass :props="props" in qth, instead it returns an error that sortable within the QTH file is undefined. 

Code: 
```             <template v-slot:header="props">
        <q-tr style="background-color: #dceefb" :props="props">
          <q-checkbox
            class="q-pl-xs"
            keep-color
            color="blue-500"
            v-model="selected_all"
            @update:model-value="selectAll"
          >
          </q-checkbox>
          <q-th v-for="col in props.cols" :key="col.name" class="text-center" :props="props">
            {{ col.label }}
          </q-th>
        </q-tr>
      </template>
```

Error: 
![image](https://user-images.githubusercontent.com/78068147/163443752-5697e865-5593-467f-b996-234c868b31b9.png)

I don't know quite how to fix this, so any help would be appreciated. Thank you!

Matthew Nguyen

**Comment by pratik227** [comment]
https://codepen.io/Pratik__007/pen/bGamOPY

**Reply by mtn090899** [reply]
Hi!

Not quite the answer I was looking for because my q-table was apart of a component, and I also wanted to include custom rows with QTR, when utilizing this solution I still got the sortable error.

However! I eventually figured out my issue, so thank you for taking a look at it!
